sap-abap/services/bdr/zcl_aws1_bdr_actions.clas.testclasses.abap

```diff
@@ -1,3 +1,5 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
 " """"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
```

Comment:
This only added the copyright instead of replacing it.

Reply:
This looks like a systemic problem so you'll need to search all of the ABAP files and remove the old comment.